### PR TITLE
[Merged by Bors] - feat: Port/Topology.Partial

### PR DESCRIPTION
**Pull request by arienmalec**
port of topology.partial

only basic naming fixes

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Partial.lean [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+open Filter
+
+open Topology
+
+variable {α β : Type _} [TopologicalSpace α]
+
+theorem rtendsto_nhds {r : Rel β α} {l : Filter β} {a : α} :
+    Rtendsto r l (𝓝 a) ↔ ∀ s, IsOpen s → a ∈ s → r.core s ∈ l :=
```
Comment (urkud) [inline Mathlib/Topology/Partial.lean:29]: Should it be called `RTendsto`?

Comment (arienmalec) [reply Mathlib/Topology/Partial.lean:29]: It's upstream in `Mathlib.Order.Filter.Partial`

In this file we have `Pcontinuous` which follows the same naming, but should be changed as a whole....

Comment (jcommelin) [reply Mathlib/Topology/Partial.lean:29]: Recorded this in #2203

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**